nautilus_trader/adapters/interactive_brokers/client/market_data.py

```diff
@@ -1117,6 +1131,8 @@ async def _process_bar_data(
             Indicates whether revised bars should be handled or not.
         historical : bool | None, optional
             Indicates whether the bar data is historical. Defaults to False.
+        start: int, optional
```

Comment:
I think if `start` is not a `pd.Timestamp` then we should always suffix the units, like `start_ns`. The choice to use nanos depends on the abstraction level, generally passing around a `pd.Timestamp` is a little nicer unless it's passing into Cython territory.